djangocms_versioning_filer/monkeypatch/admin/folderadmin.py

```diff
@@ -38,6 +37,12 @@
 from ...models import FileGrouper, get_files_distinct_grouper_queryset
 
 
```

Comment:
@jonathan-s Can we add a comment which Django versions we are targeting here, one day this will be redundant and can be cleaned out.